### PR TITLE
Feature/game socket

### PR DESCRIPTION
**Pull request by lhossackucalgary**
Update client with game state for games they're part of.
Should work with multiple tabs open of same player; 2-4 players

network heavy (sends full update every time) but this avoids some complications with reactivity in vue on vuex side.

Note: 
- Maps aren't reactive in vue 2, but board road map is inside object, so reactivity may trigger when object is updated. 

Clients should only have access to public information & their personal info (for each game).. if that's not the case, this is broken!

### CODE REVIEW

File: batan-server/app/state/game.ts [hunk 1]
```diff
@@ -168,17 +171,54 @@ export default {
...
+      board: JSON.stringify(game.gameObj.board, game.gameObj.replacer),
+      score: scores,
+      bank: {
+        resources: JSON.stringify(game.gameObj.bank.resources, game.gameObj.replacer),
+        developmentCards: dc.knight + dc.victoryPointCard + dc.roadBuilder + dc.yearOfPlenty + dc.monopoly
       }
-      callback(turnStartData);
     }
+
+    if (game.turn_num < game.players.size*2) {
```
Comment (jackadrianglass) [inline batan-server/app/state/game.ts:213]: I'm not sure I understand the * 2 part. Could you explain?

**Review by jackadrianglass — APPROVED** [review]
Looks good to me. I had one question but everything else looks great

File: batan-server/app/state/game.ts (hunk 1 above)
Comment (lhossackucalgary) [reply batan-server/app/state/game.ts:213]: The first 'turns' in the game we place 1 settlement + 1 road in forward order, then reverse - it's just checking if it's in one of those first placement turns

Comment (jackadrianglass) [reply batan-server/app/state/game.ts:213]: gotcha. Go for merge

**Pull request merged**